lib/User/UserEntry.php

```diff
@@ -211,7 +211,10 @@ public function getQuota() {
 			\OC::$server->getLogger()->debug("No LDAP quota attribute configured", ['app' => 'user_ldap']);
 		} else {
 			$quota = $this->getAttributeValue ($attr);
-			if (!$this->verifyQuotaValue($quota)) {
+			if ($quota === '') {
+				\OC::$server->getLogger()->debug("No quota set for <{$this->getOwnCloudUID()}>", ['app' => 'user_ldap']);
+				$quota = null;
+			} else if (!$this->verifyQuotaValue($quota)) {
```

Comment:
I'd prefer:
```
} else if ($quota !== null && !$this->verifyQuotaValue($quota)) {
```

Reply:
essages disapear only with @jvillafanez suggestion:

} else if ($quota !== null && !$this->verifyQuotaValue($quota)) {

without first part of "if" messages still apear.